app/scripts/modules/core/src/search/widgets/Filter.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import autoBindMethods from 'class-autobind-decorator';
+
+import { IFilterType } from './SearchFilterTypeRegistry';
+
+export interface IFilterProps {
+  filterType: IFilterType;
+  isActive: boolean;
+  onKeyUp(event: React.KeyboardEvent<HTMLElement>): void;
```

Comment:
Yikes, that was NOT what that was supposed to look like.  It should have been `onKeyUp: (event: React.KeyboardEvent<HTMLEVent>) => void`.  I didn't even realize this was valid syntax...